test/scenarios/restore/scenario.js

```diff
@@ -4,18 +4,18 @@
 
 module.exports = ({
   side: 'remote',
+  disabled:
+    "Not sure why we don't handle this case yet. At least because of init trashed property.",
   init: [
     { ino: 1, path: 'parent/' },
-    { ino: 2, path: 'parent/dir/', trashed: true },
```

Comment:
Why did you comment `trashed: true`?

Reply:
Flow was complaining about it because it is not part of the `Scenario` type.
I assumed we would figure it when working to make the scenario pass.

- [ ] Add a comment.